### PR TITLE
Merge Feature Branch to Master

### PR DESCRIPTION
**Pull request by RamseyNjire**
This is an attempt to reproduce the Newsweek with Bootstrap 4. I reproduced the main parts of the website so not all the components on the original will appear in my clone. The main aim was to recreate a believable clone and use Bootstrap to handle most of the responsiveness. I only used media queries to handle the display of the header on screens of different sizes. I hope you like it!

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes required!
Hi @RamseyNjire, well done with the project so far! :+1: However, there are some changes you need to make to complete this project. These include;
- [x] Your main navigation lacks the search input and icon and it has too much white space to the rightmost end.([Your clone](https://github.com/trekab/images/blob/master/ramsey/your_clone_main_nav.png?raw=true) Vs [Original](https://github.com/trekab/images/blob/master/ramsey/org_main_nv.png?raw=true))
- [x] Missing Humberger icon for navigation on medium and small screens. ([Your clone](https://github.com/trekab/images/blob/master/ramsey/top_story_clone.png?raw=true) Vs [Original](https://github.com/trekab/images/blob/master/ramsey/top_story_org.png?raw=true))
- [x] Featured slide show sections have too much space between images and text on the right. ([Your clone](https://github.com/trekab/images/blob/master/ramsey/slideshows_clone.png?raw=true) Vs [Original](https://github.com/trekab/images/blob/master/ramsey/slideshows_org.png?raw=true)).
- [x] `TOP STORY` should appear on top for small screens. ([Your clone](https://github.com/trekab/images/blob/master/ramsey/top_story_clone.png?raw=true) Vs [Original](https://github.com/trekab/images/blob/master/ramsey/top_story_org.png?raw=true))
- [x] `MORE STORIES` links aren't styled like the original's links.

__Kindly work on the above changes and submit another code review request.__
_Feel free to reach me on slack for inquiries about this review!_ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by RamseyNjire** [comment]
Hey, Treasure! So I fixed all the errors you pointed out, including building the missing components (searchbar and hamburger icon). A note to be made is that some spacing issues may be due to you having a 15-inch or larger screen, as the layout seems okay otherwise. I hope you like the improvements! :)

File: index.html [hunk 1]
```diff
@@ -5,33 +5,1406 @@
...
+                <a class="nav-link" href="#">Vantage</a>
+              </li>
+              <li class="nav-list-item nav-item mr-0 border-right-0">
+                <a class="nav-link" href="#">Weather</a>
+              </li>
+              <li class="nav-list nav-item d-inline-flex">
+                <form action="" class="form-inline">
+                  <input type="text" class="form-control border-0 pt-1 pb-3 pr-0 pl-0" placeholder="Search" id="searchbar">                </form>             
+                  <div class="d-inline-block mr-1" id="search-icon"></div>
+                </li>
```
Comment (lucasmazo32) [inline index.html]: - [x] Kindly fix the W3C validator error.

> image

File: index.html [hunk 2]
```diff
@@ -5,33 +5,1406 @@
...
-        </div>
-      </section>
-    </div>
+    <header>
+      <nav class="navbar sticky-top navbar-dark  pt-0 px-sm-0 px-0">
+        <div class="container-fluid py-lg-2 px-lg-1 d-flex justify-content-stretch px-0" >
 
+              <div class="col-sm-4 d-none d-md-none col-lg-3 d-sm-none d-lg-block">
+                <span class="date-display d-inline-block mt-lg-4">Mon, Mar 30, 2020</span>
+              </div>
```
Comment (lucasmazo32) [inline index.html:22]: - [x] You can set the container fluid to have a max-width of 1310px as the original page. This wa you will not have an problems with large screens.

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes listed below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by RamseyNjire** [comment]
Finished implementing the changes. Fixed the W3C validation errors as well as setting maximum width for .container-fluid to 1310 px

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @RamseyNjire, congratulations on fixing all the changes recommended by the previous TSE's :+1: On this note, I hereby approve your project. You can go ahead and merge to master and do not forget to submit the project completion form :slightly_smiling_face: 

 Good luck with your next projects
_HAPPY CODING_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**